src/Common/src/System/Number.cs

```diff
@@ -0,0 +1,1513 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Could you please move it to `src\System.Private.CoreLib\src\System\Number.cs` while you are on it to make it match CoreCLR?